src/frameworks/drupal7/composer-manager.md

```diff
@@ -0,0 +1,69 @@
+# Composer manager
+
+Drupal 7 does not natively support installing packages via Composer.  Although there is a [Drupal Composer project for Drupal 7](https://github.com/drupal-composer/drupal-project/tree/7.x), many projects are still built using a vanilla Drupal download or with Drush Make.  For sites built without Drupal Composer that still want to use modules that have Composer dependences, the most widely used option is the [Composer Manager](https://www.drupal.org/project/composer_manager) module.  Because of the read-only file system, however, there are specific configuration parameters necessary on Platform.sh.
+
+## 1. Install and patch Composer Manager
+
+Install the Composer Manager module in the manner appropriate for your site.  There are also 2 patch files needed that have not been committed to the module yet.
+
+If you're installing it via Drush Make, add the appropriate lines to your `.make` file:
+
+```ini
+projects[composer_manager][version] = "1.8"
+projects[composer_manager][patch][] = "https://www.drupal.org/files/issues/composer_manager-2620348-3.patch"
+projects[composer_manager][patch][] = "https://www.drupal.org/files/issues/composer_manager-relative_realpath-2864297-2.patch"
+```
+
+If you're checking the entire codebase into Git, do so with Composer Manager as well.  Then download and apply the two patches in the issues above and commit the result.
+
+## 2. Configure file locations
+
+Composer Manager works by using a Drush command to aggregate all module-provided `composer.json` files into a single file, which can then be installed via a normal Composer command.  Both the generated file and the resulting `vendor` directory must be in the application portion of the file system, that is, not in a writeable file mount.  As that is not the default configuration for Composer Manager it will need to be changed.  Add the following lines to your `settings.php` file:
+
+```php
+$conf['composer_manager_vendor_dir'] = '../composer/vendor';
+$conf['composer_manager_file_dir'] = '../composer';
+```
+
+The above lines will direct Composer Manager to put the generated `composer.json` file into `../composer/`, and to autoload Composer-based packages from the `../composer/vendor` directory.  The paths are relative to the Drupal root.  You may use another location if desired provided that they are not in a writable file mount, and provided the `vendor` directory is a sibling of wherever the `composer.json` file will be.
+
+Then, manually create the `composer` directory and place a `.gitignore` file inside it, containing the following, and commit it to Git:
+
+```
+# Exclude Composer dependencies.
+/vendor
+```
+
+## 3. Update the build hook
```

Comment:
Let's add a note here about adding a local `vendor` directory, at least, and note that it's caused by an upstream bug.